lib/sms_text.rb

```diff
@@ -0,0 +1,17 @@
+require 'rubygems'
+class Sms_Text
+
+	def send_text
+		account_sid = 'AC1cffe35eb134133af61d8ab716f40588'
+		auth_token = '48e47a677c14dd9162f479523ed7fb3c'
+
+		@client = Twilio::REST::Client.new account_sid, auth_token
+
+		@client.account.messages.create({
+		:from => '+442033225791',
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)